src/mudlet.pro

```diff
@@ -80,14 +80,7 @@ msvc:QMAKE_CXXFLAGS += -MP
 macx:QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.11
 
 QT += network opengl uitools multimedia gui concurrent
-qtHaveModule(gamepad) {
```

Comment:
Like in the CMake project file case - we still cover Qt 5.7 and we need to cover that older version without some modules don't we?